src/ArcGISRuntime.Samples.Shared/Models/ScreenshotSettings.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ArcGISRuntime.Samples.Shared.Models
+{
+    public class ScreenshotSettings
+    {
+        public bool ScreenshotEnabled { get; set; }
+        public string SourcePath { get; set; }
```

Comment:
This path is shared between viewers, meaning we will have to change it every time we switch between viewers. We could do separate variables or handle it at the viewer level. If we store actual `src` here, we could build the path out per platform in the per platform code.

Reply:
Good point and well spotted, I think storing the src here and building up per platform is the correct solution.